config/routes.rb

```diff
@@ -2,5 +2,6 @@
   get 'static_pages/home'
   get 'static_pages/help'
   get 'static_pages/contact'
+  get 'static_pages/about'
```

Comment:
Chuyển ' thành " nhé

Reply:
Đọc thêm convention của công ty ở link anh có gửi comment phía trên để hiểu thêm nhé, mặc định là công ty mình tránh dùng nháy đơn '' mà toàn bộ là dùng nháy kép ""
(https://github.com/framgia/coding-standards/blob/master/vn/README.md)

Reply:
Dùng nháy đôi trong ruby nhé